datadog_checks_dev/pyproject.toml

```
@@ -58,7 +58,7 @@ cli = [
     "atomicwrites",
     "beautifulsoup4>=4.9.3",
     "build>=0.7.0",
-    "click~=8.0",
+    "click==8.0",
```

```suggestion
    "click<8.1",
```
Important bug fixes were included in the dot releases before `8.1`.